lib/ansible/modules/windows/setup.ps1

```diff
@@ -307,7 +307,12 @@ if($gather_subset.Contains('memory')) {
 if($gather_subset.Contains('platform')) {
     $win32_cs = Get-LazyCimInstance Win32_ComputerSystem
     $win32_os = Get-LazyCimInstance Win32_OperatingSystem
-    $ip_props = [System.Net.NetworkInformation.IPGlobalProperties]::GetIPGlobalProperties()
+    $domainSuffix = $win32_cs.Domain.Substring($win32_cs.Workgroup.length)
```

Comment:
I would follow the existing variable standard which is `snake_case`, e.g. `$domain_suffix`.